docs/data/date-pickers/introduction/introduction.md

```diff
@@ -0,0 +1,144 @@
+---
+product: date-pickers
+title: Date and Time Picker - Introduction
+packageName: '@mui/x-date-pickers'
+githubLabel: 'component: pickers'
+materialDesign: https://m2.material.io/components/date-pickers
+waiAria: https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/datepicker-dialog.html
+---
+
+# Date and Time Pickers - Introduction
+
+<p class="description">The Date and Time pickers expose a lot of components to fit your every needs.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+## Basic usage
+
+All the components can be uncontrolled or controlled by using the `value` / `onChange` props:
+
+{{"demo": "ControlledComponent.js"}}
+
+## Imports
+
+All the components exported by `@mui/x-date-pickers` are also exported by `@mui/x-date-pickers-pro` but without the nested imports.
```

Comment:
> but without the nested imports.

What's the reason for it? Does it impact tree shaking?

Reply:
The reason is to avoid creating dozens of folder in `@mui/x-date-pickers-pro` that just re-export their equivalent from `@mui/x-date-pickers`

The tree shaking works fine on root imports in the whole MUI ecosystem.
Nested imports are only usefull for dev treeshaking.